system-tests/projects/e2e/cypress/e2e/other_target.cy.js

```diff
@@ -0,0 +1,5 @@
+context('Test', () => {
+  it('visit site with unsupported content', () => {
```

Comment:
I'm not sure we will know the intention of this test in the future without reviewing the page being loaded. Its a problem in alot of our tests IMO. A blurb or link to the issue this is solving might help give context but maybe its fine.

Reply:
Added some comments